### ISSUE TITLE
Task - Browse OTU - in the list of citations asserted distribution is always displayed as attached to the valid TaxonName OTU

### ISSUE THREAD

**Issue by proceps** [SpeciesFileGroup/taxonworks#1846]
Not sure if this bug or not.
In 3i databae (production) https://sfg.taxonworks.org/tasks/nomenclature/browse?taxon_name_id=366803
When I open asserted distribution in Browse TN (OTU radial). I have for geographic_areas. Each with one citation
![image](https://user-images.githubusercontent.com/5597748/97040602-c9515c80-1533-11eb-9127-ad85e27671d4.png)
When I open the same in OTU radial. I have much more geographic areas. Some areas with multiple citations (not sure, but some of those look like duplicate records): https://sfg.taxonworks.org/tasks/otus/browse/36540
![image](https://user-images.githubusercontent.com/5597748/97040782-13d2d900-1534-11eb-8b5d-371948169fcd.png)
When I click Expand link: I am limited to 4 geographic ares again:
![image](https://user-images.githubusercontent.com/5597748/97040993-6ca27180-1534-11eb-94c1-fd3672552854.png)

**Comment by jlpereira** [comment]
This seems to be a related with coordinate endpoint. The response contains 4 OTUs. What we should do here? merge the list?

**Comment by mjy** [comment]
@proceps Do you have records tied to synonyms?

**Comment by proceps** [comment]
Yes, I partially fooled myself. Because I started with distribution, and could not find it in Browse TN. The records are comming from the synonyms, but those has incorrect labels in Browse OTU.
![image](https://user-images.githubusercontent.com/5597748/97046877-b2176c80-153d-11eb-97df-23b20d4766db.png)

All 3 synonyms has for some reason the same OTU label (should be different, correspoding to the synonym name). Probably would also be nice to have the OTU name in the distribution table.

**Comment by mjy** [comment]
Oops. This reminds me that I though somethign was off with that list. 

* @jlpereira let's figure out why we're showing the valid OTU name not the source OTU name (likely my back-end code)

I'm not sure the best way to show OTU name in the distribution table, this was the format requested by UCD folks.  If we add OTU name then it doesn't get summarized by Geo area, and is a lot trickier to understand.  Perhaps we need an option to pivot the table to be by OTU, or have an entirely different section.

**Comment by proceps** [comment]
No, not to make it speciefic, but to add an OTU name columnt

**Comment by proceps** [comment]
and there is another error, when I click Expand hyperlink, it is restricted to a single OTU...

**Comment by proceps** [comment]
Third poind, add an radial annotator to the distribution record in the table

**Comment by mjy** [comment]
You don't need radial annotator there, but rather on an OTU summary block.

**Comment by mjy** [comment]
Good call- let's add the column, I was thinking this was the compiled summary I was working on, it's not.

**Comment by proceps** [comment]
"In general we could imagine thousands of annotors on this page- I think we have to be careful about just spamming them."
I agree

**Comment by proceps** [comment]
@mjy, it is still an issue. it looks like the issue is with the endpoint.
See an example in the 3i database
https://sfg.taxonworks.org/tasks/otus/browse/36540
The timeline has 3 OTU citation. All three related to different synonym names, but in the Timeline, the synonym names associated with OTU are displayed as valid names. you can use the radial next to each OTU to verify that each of synonyms has a single citation.
